### ISSUE TITLE
NOT an issue: a question

### ISSUE THREAD

**Issue by andypetrella** [martinring/clide2#37]
Heya Martin!

On the [Spark-Notebook](https://github.com/andypetrella/spark-notebook/) we were discussing with @virtualirfan the really awesome option to have collaboration possible in it.

Thus he came with the idea of OT, then with clide2.

I'm really really interested in this project, hence I was wondering if you had some ideas regarding an integration of a tool like a notebook, where collaboration is basically splits into cells, but of course chat and so on are still relevant.

Could we have a chat on that, would you be interested in having (or at least discussing the possibility to have) clide2 as being the OT backend for this project?

Would love to see happening.

Cheers, and congrats for clide2!

andy

**Comment by martinring** [comment]
Hey Andy,

sorry for the late reply. I was on vacation for the last two weeks. Of course I am interested. I am not too familiar with Spark though, so I might not completely understand what collaboration would look like. Are we talking about hierarchically structured data or just linear structures like text or annotated text? Because for the former clide is not (yet) suited. The core OT algorithms are also available as a subproject of clide (modules/clide-collaboration), maybe that would be more feasible to integrate than the entire infrastructure.

Cheers,
Martin.

**Comment by virtualirfan** [comment]
Hey Martin / Andy:

Thoughts on a quick Google Hangout next week?


*Irfan Ahmad*
CTO | Co-Founder | *CloudPhysics* <http://www.cloudphysics.com>
Best of VMworld Finalist
Best Cloud Management Award
NetworkWorld 10 Startups to Watch
EMA Most Notable Vendor

**Comment by martinring** [comment]
Yes sure, are you in mountain view? Then I would suggest a Hangout early in the morning maybe on thursday 9:30 AM PDT / 6:30 PM CEST?

**Comment by andypetrella** [comment]
:+1: 
Tuesday?
@martinring you're right, I think the best integration will follow the path you mentioned, that is integrating the OT libs you have developed (good job btw!)
@martinring could you send me an email to which we could send a calendar invite?

thanks mates,

cheers,
andy

**Comment by virtualirfan** [comment]
Yes, I'm in Mountain View.


*Irfan Ahmad*
CTO | Co-Founder | *CloudPhysics* <http://www.cloudphysics.com>
Best of VMworld Finalist
Best Cloud Management Award
NetworkWorld 10 Startups to Watch
EMA Most Notable Vendor
